lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1193,6 +1189,11 @@ sub register_assets_from_settings {
     # check assets and fix the file names
     for my $k (keys %assets) {
         my $a = $assets{$k};
+        if ($a->{name} =~ /\//) {
+            log_debug "not registering asset $a->{name}";
```

Comment:
It was silently returning - and now it's spitting out a debug in case you wonder what is going on.

REPO_1=ftp:// will always output this, so I don't think info or warning is good. But if you e.g. schedule ISO=fixed/mycoolone.iso and wonder why it won't it be available to workers, you would enable debug and see the debug output.